src/polar/Pie.tsx

```diff
@@ -479,6 +481,12 @@ export function computePieSectors({
         },
       ];
       const tooltipPosition = polarToCartesian(coordinate.cx, coordinate.cy, middleRadius, midAngle);
+      const outerRadiusParam = getValueByDataKey(entry, radiusKey, coordinate.outerRadius);
```

Comment:
Does this no longer allow you to specify outerRadius as you currently can? I can't tell. But it should prefer outerRadius if set - otherwise use radiusKey

Reply:
If you use both "outerRadius" and "radiusKey", the code first calculates the outer radius by using the "outerRadius" parameter. Then the "radiusKey" parameter is checked. If its provided it will use it otherwise it fallbacks to the calculated outerRadius. But i can definitely update it to prefer outerRadius parameter first.

Reply:
One concern is how a user can define a reference radius for the pie chart if the sector radius is specified as a percentage. In this case, the outer radius parameter is used to calculate the sector's radius.

Reply:
See my new comment above, I don't think this is the best way we can do this. 

If we had the prop take a number or a function we wouldn't have to prefer one over the other

Reply:
See my new comment above, I don't think this is the best way we can do this. 

If we had the prop take a number or a function we wouldn't have to prefer one over the other

Reply:
Not too sure what you mean by reference radius. Can you give me a scenario?

Reply:
i agree on making outerRadius a function. It would make more sense. Ill update it

Reply:
Have it allow both. A static number or percent like you can do today, or a function that returns a number. 

Thanks! And thanks for being willing to change things a bit

Reply:
made the changes